galaxy_importer/ansible_test/builders/local_image_build.py

```diff
@@ -0,0 +1,93 @@
+# (c) 2012-2020, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import logging
+import os
+import requests
+import subprocess
+import uuid
+
+from galaxy_importer import config
+from galaxy_importer import constants
+from galaxy_importer import exceptions
+from shutil import copy, copyfileobj, rmtree
+
+config_data = config.ConfigFile.load()
+cfg = config.Config(config_data=config_data)
+default_logger = logging.getLogger(__name__)
+
+
+class Build(object):
+    """Use docker/podman to build ansible-test image with artifact inside."""
+    def __init__(self, collection_file, collection_name, logger):
+        self.collection_file = collection_file
+        self.container_engine = Build.get_container_engine()
+        self.log = logger or default_logger
+        self.collection_name = collection_name
```

Comment:
i believe this is the collection version name rather than the collection name

Reply:
I was being consistent with https://github.com/ansible/galaxy-importer/blob/master/galaxy_importer/ansible_test/runners/local_ansible_test.py#L45.  Should I update both?

Reply:
Ah yes, staying consistent is good, and probably not worth changing the other.